apps/wallet/src/shared/sentry.ts

```diff
@@ -23,6 +23,10 @@ export default function initSentry() {
         tracesSampler: () => {
             return growthbook.getFeatureValue('wallet-sentry-tracing', 0);
         },
+        allowUrls: [
```

Comment:
Just to sanity check, you've validated these URLs? I imagine yes, they just look so plain that it feels wrong, but I'm sure it's right.